### PR TITLE
The size of rect can equal to the size of image when cropping image.

### PR DESCRIPTION
**Pull request by dirtmelon**
## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.0.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Comment by guykogus** [comment]
Hey @dirtmelon, thanks for submitting this PR. Can you please add a changelog entry and update any test cases (if possible)? Cheers :)

**Comment by dirtmelon** [comment]
@guykogus I will add a changelog entry and update test cases :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ### Changed
 - **UIApplication**:
   - - `queryValue(for:)` extension for URL is refactored. [#668](https://github.com/SwifterSwift/SwifterSwift/pull/668) by [LucianoPAlmeida](https://github.com/ratulSharker).
-
+- **UIImage**:
```
Comment (guykogus) [inline CHANGELOG.md]: I think that this should be in the `Fixed` section, rather than `Changed`.

**Comment by LucianoPAlmeida** [comment]
We are having issues with out build, so it fails because we are having issues with `pod lib lint`, needs futher investigation. I'm removing that step in #677 for now so we can have the builds passing on PR's and then we do investigate it in parallel :))

**Comment by SD10** [comment]
Thank you for contributing to SwifterSwift! I've invited you to join the SwifterSwift GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md). Feel free to reach out if you have any questions! 😃

**Pull request merged**